lms/static/scripts/frontend_apps/components/Table.js

```diff
@@ -100,7 +100,12 @@ export default function Table({
   };
 
   return (
-    <div className="Table__wrapper">
+    <div
+      className={classnames({
+        Table__wrapper: true,
+        'has-files': items.length,
```

Comment:
I'm guessing that any truthy value produces the same result here, though I personally prefer using a comparison `items.length > 0` to make this more explicit. We don't have a firm convention within the team.